### ISSUE TITLE
Underline is not displayed correctly

### ISSUE THREAD

**Issue by Azathoth88** [dopplertask/dopplertask#29]
If I use underscores in the MySQL action (e.g. user or password), this is not displayed correctly in the browser. The input field is too narrow. Example: https://i.imgur.com/GrS361T.png

**Comment by feraswilson** [comment]
Yes, I am aware of this "issue". I have to dig a bit more to see how a better approach would be.

**Comment by Azathoth88** [comment]
Try to increase the padding of the css-class .ace_editor with 10px
